### PR TITLE
Add release notes 62; other misc updates

### PR DESCRIPTION
**Pull request by erkangz**

### CODE REVIEW

File: release-notes/2021-62.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+- Another significant query performance improvement for all types of queries
+  (including aggregate queries, queries with negations, and optional
+  traversals).
+
+- Added _Cursor_ support for J1QL queries via the API. 
+
+  See details in the **Querying the graph with J1QL** section of the 
+  [Platform API Doc](../docs/jupiterone-api.md).
+
+- Visual Query Builder now supports aggregate returns (i.e. `COUNT`, `SUM`,
```
Comment (henrymgarrett) [inline release-notes/2021-62.md:41]: VQB also now supports:
1. Grouping returns by an attribute value
2. Allowing filter values to be selected as `undefined`

File: release-notes/2021-62.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+- Fixed an issue that may cause the Compliance app to trigger an infinite load screen
+
+- Fixed an issue where a user could not update the audit tracking field for a compliance item via the UI
+
+- Fixed an issue where the graph view would refresh if an entity was moved
+
+- Added sorting to the rule table in the alerts app
+
+- Compliance reviews can now be fully deleted
```
Comment (henrymgarrett) [inline release-notes/2021-62.md:126]: Last sprint, @byricketts also added the ability to assign compliance review owners at the standard level (previously this was not supported in the UI)

Comment (henrymgarrett) [reply release-notes/2021-62.md:126]: Related to compliance reviews - the user assignment dropdown shown in the `Edit Review Configuration` modal now shows names, avatars AND emails to more accurately identify user accounts.  Again, this was @byricketts

Comment (henrymgarrett) [reply release-notes/2021-62.md:126]: And Compliance Reviews is now GA

**Review by henrymgarrett — APPROVED** [review]

File: release-notes/2021-62.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+  Check out the [docs](https://github.com/JupiterOne/graph-malwarebytes/blob/master/docs/jupiterone.md) 
+  for details on what's currently supported.
+
+### Qualys
+
+- Duplicate values in Finding.targets are removed
+- Fixed memory leak impacting long running instances
+- Advanced the mock Qualys server to facilitate load testing
+- Handling invalid `DETECTION.QID` values such as `'{"#text":"o��t�","��\u001dX�Ď��,</QID'`
```
Comment (alee-j1) [inline release-notes/2021-62.md:106]: @erkangz Are those question marks in the code text supposed to render like that?

**Review by alee-j1 — APPROVED** [review]
Just one comment on a bullet under Qualys. Other than that, looks good to me!

**Pull request merged**